lib/types.ts

```diff
@@ -0,0 +1,27 @@
+import ClientOAuth2 = require("client-oauth2");
+
+export type ClientOpts = {
+  clientId: string;
+  baseUrl?: string;
+  redirectUri: string;
+  scopes: string[];
+};
+
+export type GetAuthUriOpts = {
+  codeChallenge?: string;
+  codeChallengeMethod?: string;
+};
+
+export type GetTokenOpts = {
+  codeVerifier?: string;
+};
+
+export declare class Client {
+  private oauth2Client: ClientOAuth2;
+  private state: string;
+
+  constructor(opts: ClientOpts);
+
+  public getAuthUri(opts: GetAuthUriOpts): Promise<string>;
+  public getToken(callbackUri: string, opts: GetTokenOpts): Promise<ClientOAuth2.Token>;
+}
```

Comment:
We do not need this. Just run `tsc` and look into `dist/client.d.ts` which will create sth. similar already.

Reply:
true, corrected with https://github.com/kontist/sdk/pull/1/commits/4f75a79b29463f3c788741cfe8fe8157d4dcfa84